src/main/scala/intellij/haskell/module/HaskellModuleBuilder.scala

```diff
@@ -174,7 +175,7 @@ class HaskellModuleWizardStep(wizardContext: WizardContext, haskellModuleBuilder
 
 object HaskellModuleBuilder {
 
-  private final val LibName = ".ideaHaskellLib"
+  private final val LibName = s".intellij-haskell${File.separator}lib"
```

Comment:
Because the name of `library root` can not be set manually, it will display the name of the last child directory, and I think `lib` is better than `.ideaHaskellLib` showing there, so I changed `LibName`, just for displaying `lib` in library tree under SDK.

![image](https://cloud.githubusercontent.com/assets/6234553/22342266/bcddf466-e42e-11e6-9fb6-f596ac9d7ae1.png)